sdk/storage/Azure.Storage.Common/tests/Shared/ClientSideEncryptionTestExtensions.cs

```diff
@@ -177,8 +177,41 @@ public static Mock<Microsoft.Azure.KeyVault.Core.IKeyResolver> GetTrackOneIKeyRe
         private static byte[] KeyWrapImpl(byte[] key, byte[] contents)
         {
             var result = new byte[contents.Length];
-            // just bitflip the contents
-            new System.Collections.BitArray(contents).Not().CopyTo(result, 0);
+
+            if (contents.Length == 0)
+            {
+                return result;
+            }
+
+            // Move each byte one position to the left
+            for (int i = 0; i < contents.Length - 1; i++)
+            {
+                result[i] = contents[i + 1];
+            }
```

Comment:
```csharp
new Span<byte>(key, 1).CopyTo(result);
```
No need to write a whole for loop here. An array copy is more readable. This will require also special-casing length = 1 but imo worth it.